Packs/AWS-SNS/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "AWS - SNS",
+    "description": "This is the integration content pack which can create or delete topic/subscription on AWS Simple Notification System and send the message via SNS as well.",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Jie Liau",
+    "url": "",
+    "email": "",
+    "created": "2022-05-25T03:52:46Z",
+    "categories": [],
+    "tags": [],
```

Comment:
according the yml file - the `categories ` should be "IT Services"

see Packs/AWS-ACM/pack_metadata.json as reference

Reply:
@ilappe Added